src/lib/request-with-rate-limit.ts

```diff
@@ -23,6 +23,7 @@ export async function limiterWithRateLimitRetries<ResponseType>(
   };
   const maxRetries = 7;
   let attempt = 0;
+  url = encodeURI(url);
```

Comment:
(not a blocker) would be nicer to do [inline here](https://github.com/snyk-tech-services/snyk-api-import/pull/296/files#diff-51c7d2f88e2710cc7177b815407d82104b536a1274aec6e52da65e9d03c58878R38) or as a new variable, data mutations are hard to follow usually